src/vn200_node.cpp

```diff
@@ -464,20 +467,33 @@ int main(int argc, char* argv[])
     pub_gps     = n_.advertise<vectornav::gps>    ("gps", 1000);
     pub_sensors = n_.advertise<vectornav::sensors>("imu", 1000);
 
+
     // Initialize VectorNav
     //VN_ERROR_CODE vn_retval;
     ROS_INFO("Initializing vn200. Port:%s Baud:%d\n", port.c_str(), baud);
 
     try {
         vn200.connect(port, baud);
     } catch (...) {
-        ROS_FATAL("Could not conenct to vn200 on port:%s @ Baud:%d;"
+        ROS_FATAL("Could not connect to vn200 on port:%s @ Baud:%d;"
                 "Did you add your user to the 'dialout' group in /etc/group?", 
                 port.c_str(), 
                 baud); 
         exit(EXIT_FAILURE);
     }
 
+    ROS_INFO("Initializing vn200_2. Port:%s Baud:%d\n", "/dev/ttyTHS1", baud);
+
+    try {
+        vn200_2.connect("/dev/ttyTHS1", baud);
```

Comment:
@neelfirst FYI: not everyone will have a ttyTHS1 device, so this is kinda bad form to put this in a library that will be used by the public. feel free to fix it in your next PR or not, I'm sure someone will at some point. :)